docs/source/access.mdx

```diff
@@ -1,126 +1,81 @@
 
 ```py
+# Get the first row in the dataset
 >>> dataset[0]
-{'idx': 0,
-'label': 1,
-'sentence1': 'Amrozi accused his brother , whom he called " the witness " , of deliberately distorting his evidence .',
-'sentence2': 'Referring to him as only " the witness " , Amrozi accused his brother of deliberately distorting his evidence .'}
+{'label': 1,
+ 'text': 'the rock is destined to be the 21st century\'s new " conan " and that he\'s going to make a splash even greater than arnold schwarzenegger , jean-claud van damme or steven segal .'}
 ```
 
-Return the number of rows and columns with the following standard attributes:
+Use the `-` operator to start from the end of the dataset:
 
 ```py
->>> dataset.shape
-(3668, 4)
->>> dataset.num_columns
-4
->>> dataset.num_rows
-3668
->>> len(dataset)
-3668
+# Get the last row in the dataset
+>>> dataset[-1]
+{'label': 0,
+ 'text': 'things really get weird , though not particularly scary : the movie is all portent and no content .'}
 ```
 
-List the columns names with [`Dataset.column_names`]:
+Indexing by the column name returns a list of all the values in the column:
 
 ```py
->>> dataset.column_names
-['idx', 'label', 'sentence1', 'sentence2']
+>>> dataset["text"]
+['the rock is destined to be the 21st century\'s new " conan " and that he\'s going to make a splash even greater than arnold schwarzenegger , jean-claud van damme or steven segal .',
+ 'the gorgeously elaborate continuation of " the lord of the rings " trilogy is so huge that a column of words cannot adequately describe co-writer/director peter jackson\'s expanded vision of j . r . r . tolkien\'s middle-earth .',
+ 'effective but too-tepid biopic',
+ ...,
+ 'things really get weird , though not particularly scary : the movie is all portent and no content .']
 ```
 
-Get detailed information about the columns with [`~datasets.Features`]:
+You can combine row and column name indexing to return a specific value at a position:
 
 ```py
->>> dataset.features
-{'idx': Value(dtype='int32', id=None),
-    'label': ClassLabel(num_classes=2, names=['not_equivalent', 'equivalent'], names_file=None, id=None),
-    'sentence1': Value(dtype='string', id=None),
-    'sentence2': Value(dtype='string', id=None),
-}
+>>> dataset[0]["text"]
+'the rock is destined to be the 21st century\'s new " conan " and that he\'s going to make a splash even greater than arnold schwarzenegger , jean-claud van damme or steven segal .'
 ```
 
-Return even more specific information about a feature like [`ClassLabel`], by calling its parameters `num_classes` and `names`:
+But it is important to remember that indexing order matters, especially when you're working with large or audio and image datasets. Indexing by the column name returns all the values in the column first, then loads the value at that position. For large datasets, it may be slower to index by the column name first.
```

Comment:
```suggestion
But it is important to remember that indexing order matters, especially when working with large audio and image datasets. Indexing by the column name returns all the values in the column first, then loads the value at that position. For large datasets, it may be slower to index by the column name first.
```